hr_contract_amendment/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+=====================
+HR CONTRACT AMENDMENT
+=====================
+
+This module extends the functionality of HR contract to keep history of change
+and to allow you to saw when and who made contract change
+
+Installation
+============
+
+To install this module, you need to:
+
+ just install it
```

Comment:
Not all persons are technical, so expand this with a standard instalation steps:
+To install this module, you need to:
+
+* clone the branch 11.0 of the repository https://github.com/OCA/hr
+* add the path to this repository in your configuration (addons-path)
+* update the module list
+* search for "HR Contract Amendment" in your addons
+* install the module

Reply:
This obvious instructions should be omitted I guess at all, unless there are any module-specific steps